### PR TITLE
add virtualbox v5.0.0-beta2

### PR DESCRIPTION
**Pull request by sandlerr**

### CODE REVIEW

**Comment by sandlerr** [comment]
cc @vitorgalvao any problems I need to fix here?

**Comment by vitorgalvao** [comment]
Apologies, as it slipped through. Yes, there’s just a tiny change needed (will comment on the line).

After you do it, please do not forget to [squash your commits](http://davidwalsh.name/squash-commits-git), and @mention me so I get a warning to come back.

File: Casks/virtualbox5.0.0-beta2-99639.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'virtualbox5.0.0-beta2-99639' do
+  version '5.0.0-beta2-99639'
+  sha256 '3ed03339ba395900ef01824c86e14657a4e5d4aec1c630172e7541b6c585d49a'
+
+  url "http://dlc-cdn.sun.com/virtualbox/5.0.0_BETA2/VirtualBox-5.0.0_BETA2-99639-OSX.dmg"
```
Comment (vitorgalvao) [inline Casks/virtualbox5.0.0-beta2-99639.rb]: Needs a `# sun.com is the official download host per the vendor homepage` comment above this line ([example](https://github.com/caskroom/homebrew-cask/blob/cb487ad405f74cb1f26b93e3f3e8412c82a22e34/Casks/0ad.rb#L5)).

**Comment by vitorgalvao** [comment]
Superseded by #956.

**Pull request closed without merging**